README.md

```diff
@@ -130,6 +130,27 @@ end
 The `setup` is run once on each worker process that requires it;
 it is not run before every `@testitem` that depends on the setup.
 
+#### Skipping tests
+
+The `skip` keyword can be used to skip a `@testitem`, meaning no code inside that test-item will run.
+A skipped test-item logs that it is being skipped and records a single "skipped" test result, similar to `@test_skip`.
+
+```julia
+@testitem "skipped" skip=true begin
+    @test false
+end
+```
+
+If `skip` is an given an `Expr`, it must return a `Bool` indicating whether or not to skip the test-item.
+This expression will be run in a new module similar to a test-item immediately before the test-item would be run.
```

Comment:
```suggestion
If `skip` is given as an `Expr`, it must return a `Bool` indicating whether or not to skip the test-item.
This expression will be run in a new module similar to a test-item immediately before the test-item would be run.
```

Reply:
technically I guess this should be ```must evaluate to a `Bool` ```, rather than `return`, but I think maybe the way you have it is easier to understand?

Reply:
yeah, talk to `evaluate` has been griped about https://github.com/JuliaTesting/ReTestItems.jl/commit/0b7275325db58eba956e1d23a375a81fc34a8efe 😆